usc_dae_utils/operations.py

```diff
@@ -0,0 +1,104 @@
+import random
```

Comment:
I guess that's the only file we need, no?

Reply:
2c48a68